packages/client3/components/design-system/input/checkbox.tsx

```diff
@@ -1,76 +1,75 @@
 import clsx from "clsx";
 import { forwardRef, InputHTMLAttributes, ReactNode } from "react";
 
+import { HelperText } from "../typography";
```

Comment:
This will need refactor once the typography is merged.

Reply:
Yeah sure